### DISCUSSION TITLE
Is it possible to combine examples?

### DISCUSSION THREAD

**Discussion by kamelkeb** [vercel/next.js#12365] [Help] [answered]
Hi, 
Not familiar with Next.js, willing to use it professionally in the short future, I wanted to have an example using antd with less and jest for testing. Looking at the way examples are named, a lot of them is prefixed by the world "with" so it reminds me of traits that can be combiened.
Knowing that to use the example with less and antd  the command
`npm init next-app --example with-ant-design-less with-ant-design-app`
will do the trick, and to use the example with jest the command
`npx create-next-app --example with-jest`
does the job.
Will the command 
`npm init next-app --example with-ant-design-less with-ant-design-app with-jest`
provide me an example of well reconfigured project that uses antd, less and jest?
Thank you

**Comment by rafaelalmeidatk** [comment, marked as answer]
It is not possible to combine examples, what you can do is look at all the examples you like and copy their content into your own project

**Reply by kamelkeb** [reply]
Thank you!

**Reply by kayoroxa** [reply]
maybe it can help you: https://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories

**Comment by kayoroxa** [comment]
maybe it can help you: https://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories
